lib/WebpackOptionsDefaulter.js

```diff
@@ -258,6 +258,8 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			"make",
 			options => options.mode === "development"
 		);
+		// TODO enable for production mode in webpack 5
```

Comment:
Remove this comment. I don't think it make sense to have this on by default.

Reply:
@sokra for better long term caching?

Reply:
2 reasons:

Currently the production mode don't enable long term caching by default. That's a bit difficult to do without control over the HTML files (and the server configuration). And stable ids is not relevant for everyone. If you deploy rarely and/or have a small application you may need smaller bundles more than stable ids.

Hashed modules is only the second best solution for long term caching. Records is the better option. It creates smaller bundles. Enabling it by default would mean to have to disable it to use the best solution, making using the best solution more difficult.

Reply:
@sokra looks, we should documented this :smile: Where we can find example?